### ISSUE TITLE
gitlab_user_enum RuntimeError

### ISSUE THREAD

**Issue by xl7dev** [rapid7/metasploit-framework#7535]
```
msf auxiliary(gitlab_user_enum) > show options

Module options (auxiliary/scanner/http/gitlab_user_enum):

   Name       Current Setting      Required  Description
   ----       ---------------      --------  -----------
   END_ID     50                   yes       ID number to enumerate up to
   Proxies                         no        A proxy chain of format type:host:port[,type:host:port][...]
   RHOSTS     192.168.0.100         yes       The target address range or CIDR identifier
   RPORT      80                   yes       The target port
   SSL        false                no        Negotiate SSL/TLS for outgoing connections
   START_ID   0                    yes       ID number to start from
   TARGETURI  /                    yes       Path to GitLab instance
   THREADS    1                    yes       The number of concurrent threads
   VHOST      gitlab.example.com  no        HTTP server virtual host

msf auxiliary(gitlab_user_enum) > run

[*] Sending GitLab version request...
[-] Auxiliary failed: RuntimeError not-vulnerable: Unable to retrieve GitLab version...
[-] Call stack:
[-]   /opt/metasploit-framework/embedded/framework/lib/msf/core/module.rb:290:in `fail_with'
[-]   /opt/metasploit-framework/embedded/framework/modules/auxiliary/scanner/http/gitlab_user_enum.rb:77:in `run_host'
[-]   /opt/metasploit-framework/embedded/framework/lib/msf/core/auxiliary/scanner.rb:135:in `block (2 levels) in run'
[-]   /opt/metasploit-framework/embedded/framework/lib/msf/core/thread_manager.rb:100:in `call'
[-]   /opt/metasploit-framework/embedded/framework/lib/msf/core/thread_manager.rb:100:in `block in spawn'
[*] Auxiliary module execution completed
```

**Comment by wvu-r7** [comment]
You sure it's GitLab?

**Comment by wvu-r7** [comment]
Maybe your ```TARGETURI``` is wrong?

**Comment by h00die** [comment]
edited ticket to wrap in code execution so it looks correct

**Comment by nixawk** [comment]
@xl7dev everything works good. I guess that your target is not a gitlab site.

```
msf auxiliary(gitlab_user_enum) > show options

Module options (auxiliary/scanner/http/gitlab_user_enum):

   Name       Current Setting  Required  Description
   ----       ---------------  --------  -----------
   END_ID     50               yes       ID number to enumerate up to
   Proxies                     no        A proxy chain of format type:host:port[,type:host:port][...]
   RHOSTS     www.rapid7.com   yes       The target address range or CIDR identifier
   RPORT      80               yes       The target port
   SSL        false            no        Negotiate SSL/TLS for outgoing connections
   START_ID   0                yes       ID number to start from
   TARGETURI  /                yes       Path to GitLab instance
   THREADS    1                yes       The number of concurrent threads
   VHOST      www.rapid7.com   no        HTTP server virtual host

msf auxiliary(gitlab_user_enum) > run

[*] Sending GitLab version request...
[-] Auxiliary failed: RuntimeError unknown: Unable to retrieve GitLab version...
[-] Call stack:
[-]   /Users/Open-Security/Code/metasploit-framework/lib/msf/core/module.rb:290:in `fail_with'
[-]   /Users/Open-Security/Code/metasploit-framework/modules/auxiliary/scanner/http/gitlab_user_enum.rb:79:in `run_host'
[-]   /Users/Open-Security/Code/metasploit-framework/lib/msf/core/auxiliary/scanner.rb:135:in `block (2 levels) in run'
[-]   /Users/Open-Security/Code/metasploit-framework/lib/msf/core/thread_manager.rb:100:in `block in spawn'
[*] Auxiliary module execution completed
```

**Comment by wvu-r7** [comment]
Closing this, since it's not a bug. Feel free to reach out in the usual support channels.

**Comment by xl7dev** [comment]
@wvu-r7 @h00die  I'm sorry,This issue was fixed in GitLab v7.5.0 and is present from
  GitLab v5.0.0,

**Comment by wvu-r7** [comment]
@xl7dev: Okay.
